upgrading.md

```diff
@@ -0,0 +1,36 @@
+Upgrading OpenSearch Python Client
+==================================
+
+### Upgrading to >= 2.2.0
+
+Opensearch-py now includes opensearch-dsl-py features. Opensearch-dsl-py was merged into opensearch-py in the following PR [#287](https://github.com/opensearch-project/opensearch-py/pull/287). 
+
+
+#### Overview of architectural changes:
+
+Opensearch-py functionalities that already exist are not altered. Only the following areas have new additions.
+
+The opensearchpy "helpers" module now provides access to aggs, analysis, document, faceted search, field, function, index, mapping, query, search, update by query, utils, and wrappers. As a result, without importing opensearch-dsl-py, these functionalities can be imported directly from opensearch-py. The "connections" file from DSL is now included in the "connection" module. Additionally, the dsl files "serializer" and "exceptions" are combined with their equivalent opensearchpy files "serializer" and "exceptions".
+
+
+#### API Implementation differences for existing opensearch-dsl-py users 
+
+As mentioned above, Opensearch-py functionalities that already exist are not altered. Switching from opensearch-dsl-py to opensearch-py is made simple by minimising API implementation differences. Refer to the [userguide](https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md#using-high-level-python-client) for an example of how to implement a newly added feature.
```

Comment:
We can say "The functionalities from opensearch-dsl-py are merged into this client. Refer to the [USER_GUIDE](https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md#using-high-level-python-client) for an example on how to implement a feature from the previously known high level Python client."